charts/fluid-datamigrate/juicefs/templates/cronjob.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       backoffLimit: {{ .Values.datamigrate.backoffLimit | default "3" }}
       completions: 1
       parallelism: 1
+      {{- if gt (.Values.datamigrate.parallelism | int) 1 }}
+      # when using parallel tasks, default suspend is true, the reconciler will set it to false after scale the workers statefulset.
+      suspend: true
```

Comment:
Which Kubernetes version can support this?

Reply:
job suspend field release notes: https://github.com/kubernetes/enhancements/issues/2232

Alpha release target (x.y): 1.21, need explictly enable the `SuspendJob` feature gate;
Beta release target (x.y): 1.22, gated by SuspendJob feature flag (enabled by default);
Stable release target (x.y): 1.24

Reply:
So I think we should stress the known limitations in the documentation. The endusers could be aware of this.

Reply:
fixed.